integration_tests/k8s/k8sclient.go

```diff
@@ -0,0 +1,72 @@
+package k8s
```

Comment:
can we have a client common client package so that we can use it in both ndm code and it.

Reply:
@imazik , currently we will keep the BDD code and NDM business logic code separate. If they converge on the same methods, then we will move into `pkg`